sling/nlp/parser/trainer/transition-generator.h

```diff
@@ -0,0 +1,37 @@
+#ifndef SLING_NLP_PARSER_TRAINER_TRANSITION_GENERATOR_H_
```

Comment:
You are missing the copyright header here.

Reply:
Done.

Reply:
Done.